src/core/telemetry/tcp_tracer.h

```diff
@@ -108,6 +108,8 @@ struct TcpConnectionMetrics {
   std::optional<uint32_t> rcvq_drops;
   // The NIC Rx delay reported by the remote host.
   std::optional<uint32_t> nic_rx_delay_usec;
+
+  std::string ToString();
```

Comment:
Just to note directionally I think we agreed this type is deprecated and should be removed, in favor of the more extensible query mechanism added to `EventEngine::Endpoint` in https://github.com/grpc/grpc/pull/39277.

Reply:
ah you are right, these are not needed anymore. I can directly use those instead of converting the metrics from EventEngine::Endpoint to this.